lstchain/io/lstcontainers.py

```diff
@@ -178,3 +179,21 @@ class DispContainer(Container):
     norm = Field(nan, 'Norm of the disp_norm vector')
     sign = Field(nan, 'Sign of the disp_norm')
     miss = Field(nan, 'miss parameter norm')
+
+class DriveReport(Container):
```

Comment:
A class is created and not used anywhere?

Reply:
Ok I see this is related to PR #153 .

Reply:
No, it will be used. I'll upload another script soon which uses this class.

Reply:
Why not directly including this in PR #153 ?
This PR makes little sense on its own.

Reply:
> No, it will be used. I'll upload another script soon which uses this class.

Then please do it all at once to help the review process.
One cannot judge the relevance of this code with anything using it.

Reply:
Actually, this can be part of ctapipe as well.

Reply:
Even better.